tests/unit/controllers/service_test.js

```diff
@@ -0,0 +1,34 @@
+import ServiceController from 'js/controllers/service';
+import Service from 'js/views/service';
+
+describe('Service controller tests', function () {
+  let foxboxStub, mountNodeStub, controller;
+
+  before(function() {
+    foxboxStub = sinon.stub();
+    mountNodeStub = document.createElement('node');
+
+    controller = new ServiceController({
+      foxbox: foxboxStub,
+      mountNode: mountNodeStub
+    });
+  });
+
+  it('Controller is successfully created', function() {
+    assert.isDefined(controller);
+  });
+
+  it('Renders view with correct parameters', function () {
```

Comment:
As discussed, that's perfect for now, we need the tooling first then we can add unit tests as we go.